### PR TITLE
[MODULAR]Mold event tweaks

### PR DESCRIPTION
**Pull request by Vizoplez**
## About The Pull Request
Mold is undoubtedly quite obnoxious during lowpop/other times where dynamic is hesitant to spawn antagonists; resulting in threat building up and mold potentially proccing multiple times per round. This fixes that, reduces mold weight and max occurances, and disables cordyceps until its reworked.

## Why It's Good For The Game
Mass disease spam + 2 other cores to deal with during this is never fun.

## Changelog
:cl:
tweak: Mold weight & Max occurances
tweak: Mold rats now respect hardsuits.
/:cl:

### CODE REVIEW

**Review by Gandalf2k15 — APPROVED** [review]

File: modular_skyrat/modules/biohazard_blob/code/mold_event.dm [hunk 1]
```diff
@@ -39,7 +46,11 @@
 	qdel(resintest)
 
 	for(var/i = 1, i <= molds2spawn)
-		var/picked_mold = pick(available_molds)
+		var/picked_mold
+		if(get_active_player_count(alive_check = TRUE, afk_check = TRUE, human_check = TRUE) >= 60)
+			picked_mold = pick(available_molds_t2)
```
Comment (NotRanged) [inline modular_skyrat/modules/biohazard_blob/code/mold_event.dm:51]: consider pick_n_take for guaranteed variety perhaps

Comment (NotRanged) [reply modular_skyrat/modules/biohazard_blob/code/mold_event.dm:51]: eh already merged

**Comment by FlamingLily** [comment]
man, I really enjoyed that cordyceps round, actually, kinda sucks to see it go

File: modular_skyrat/modules/biohazard_blob/code/biohazard_blob_mobs.dm [hunk 2]
```diff
@@ -96,7 +96,7 @@
 	. = ..()
 	if(iscarbon(target))
 		var/mob/living/carbon/C = target
-		if(prob(40))
+		if(src.can_inject(target))
```
Comment (KasparoVy) [inline modular_skyrat/modules/biohazard_blob/code/biohazard_blob_mobs.dm:99]: Guaranteed noms unless wearing a suit though :o

**Pull request merged**